pkg/kv/kvserver/stateloader/stateloader.go

```diff
@@ -80,14 +80,6 @@ func (rsl StateLoader) Load(
 	s.Stats = &ms
 	s.RaftClosedTimestamp = as.RaftClosedTimestamp
 
```

Comment:
Can explicitly say `TruncatedState = nil` here, to stress that this field is unused and should be loaded separately.